### ISSUE TITLE
503 for master

### ISSUE THREAD

**Issue by dblooman** [beevelop/docker-directus#20]
Hi,

On starting docker compose, all I can see is 503's

some output

``` sh
directus_1  | Running autoconfig script...
directus_1  |
directus_1  | Warning: require(/var/www/html/api/config.php): failed to open stream: No such file or directory in /var/www/html/bin/directus on line 7
directus_1  |
directus_1  | Fatal error: require(): Failed opening required '/var/www/html/api/config.php' (include_path='.:/usr/local/lib/php') in /var/www/html/bin/directus on line 7
directus_1  | Running autoconfig script...
directus_1  |
directus_1  | Warning: require(/var/www/html/api/config.php): failed to open stream: No such file or directory in /var/www/html/bin/directus on line 7
directus_1  |
directus_1  | Fatal error: require(): Failed opening required '/var/www/html/api/config.php' (include_path='.:/usr/local/lib/php') in /var/www/html/bin/directus on line 7
proxy_1     | forego     | starting nginx.1 on port 5000
proxy_1     | forego     | starting dockergen.1 on port 5100
dockerdirectus_volumes_1 exited with code 0
proxy_1     | dockergen.1 | 2016/09/12 10:17:27 Generated '/etc/nginx/conf.d/default.conf' from 3 containers
proxy_1     | dockergen.1 | 2016/09/12 10:17:27 Watching docker events
proxy_1     | dockergen.1 | 2016/09/12 10:17:27 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification 'nginx -s reload'
directus_1  | Running autoconfig script...
proxy_1     | dockergen.1 | 2016/09/12 10:17:34 Received event start for container d3564ac82b7d
proxy_1     | dockergen.1 | 2016/09/12 10:17:34 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification 'nginx -s reload'
directus_1  |
directus_1  | Warning: require(/var/www/html/api/config.php): failed to open stream: No such file or directory in /var/www/html/bin/directus on line 7
directus_1  |
directus_1  | Fatal error: require(): Failed opening required '/var/www/html/api/config.php' (include_path='.:/usr/local/lib/php') in /var/www/html/bin/directus on line 7
proxy_1     | dockergen.1 | 2016/09/12 10:17:34 Received event die for container d3564ac82b7d
proxy_1     | dockergen.1 | 2016/09/12 10:17:35 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification 'nginx -s reload'
proxy_1     | nginx.1    | localhost 172.17.0.1 - - [12/Sep/2016:10:17:41 +0000] "GET / HTTP/1.1" 503 615 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36"
proxy_1     | nginx.1    | localhost 172.17.0.1 - - [12/Sep/2016:10:17:42 +0000] "GET / HTTP/1.1" 503 615 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36"
proxy_1     | nginx.1    | localhost 172.17.0.1 - - [12/Sep/2016:10:17:43 +0000] "GET / HTTP/1.1" 503 615 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36"
proxy_1     | nginx.1    | localhost 172.17.0.1 - - [12/Sep/2016:10:17:43 +0000] "GET / HTTP/1.1" 503 615 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36"
proxy_1     | nginx.1    | localhost 172.17.0.1 - - [12/Sep/2016:10:17:54 +0000] "GET / HTTP/1.1" 503 615 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36"
directus_1  | Running autoconfig script...
proxy_1     | dockergen.1 | 2016/09/12 10:18:26 Received event start for container d3564ac82b7d
proxy_1     | dockergen.1 | 2016/09/12 10:18:26 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification 'nginx -s reload'
directus_1  |
directus_1  | Warning: require(/var/www/html/api/config.php): failed to open stream: No such file or directory in /var/www/html/bin/directus on line 7
directus_1  |
directus_1  | Fatal error: require(): Failed opening required '/var/www/html/api/config.php' (include_path='.:/usr/local/lib/php') in /var/www/html/bin/directus on line 7
proxy_1     | dockergen.1 | 2016/09/12 10:18:26 Received event die for container d3564ac82b7d
proxy_1     | dockergen.1 | 2016/09/12 10:18:26 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification 'nginx -s reload'
```

**Comment by beevelop** [comment]
Unfortunately this is caused by breaking changes to Directus' CLI installation and apparently major bugs as side-effects. You might need to stick to version 6.2.8 until the underlying causes are resolved. I'll keep you updated

**Comment by dblooman** [comment]
@beevelop thanks, i ended up going the native route, which was a pain.  Still interested in use Docker and Compose.

**Comment by beevelop** [comment]
You can track the discussion of the underlying issue over here: directus/directus#1259

**Comment by beevelop** [comment]
This issue is resolved with Directus' latest release: v6.3.4 (see #21, #22)
